example-server.js

```diff
@@ -95,12 +95,14 @@ function concatProof(existingProof, newProof) {
 }
 
 function validateUrl(value) {
-  if(value.startsWith('urn:')) {
-    return 'URN is not a URL';
-  }
   if(value.indexOf(':') === -1) {
```

Comment:
Could use `includes` for this.  Is this check and error needed rather than the more generic `ERR_INVALID_URL` `TypeError` that would be caught below?
```suggestion
  if(!value.includes(':')) {
```

Reply:
This is more of a commentary on @clehner 's code and not part of this PR which turns the URNs to invalid urls. Maybe you should make an issue with code suggestions and code clean up for this project after the lint PR lands?